source/localizable/components/triggering-changes-with-actions.md

```diff
@@ -245,76 +245,90 @@ In this case, the code in `button-with-confirmation` does not change.
 It will still invoke `onConfirm` with no arguments.
 The action helper will add the arguments provided in the template to the call.
 
-Action arguments curry, meaning that you can provide partial arguments to the action helper and provide the rest of the
-arguments when you call the function within the component javascript file.
-For example, our `button-with-confirmation` component will now [yield](../wrapping-content-in-a-component/) the content
-of the confirmation dialog to collect extra information to be sent along with the `onConfirm` action:
+So far in our example,
+the action we have passed to `button-with-confirmation` is a function that accepts one argument, `messageType`.
+Suppose we want to extend this by allowing `sendMessage` to take a second argument,
+the actual text of the message the user is sending:
 
-```app/templates/components/button-with-confirmation.hbs
-<button {{action "launchConfirmDialog"}}>{{text}}</button>
-{{#if confirmShown}}
-  <div class="confirm-dialog">
-    {{yield confirmValue}}
-    <button class="confirm-submit" {{action "submitConfirm"}}>OK</button>
-    <button class="confirm-cancel" {{action "cancelConfirm"}}>Cancel</button>
-  </div>
-{{/if}}
+```app/components/send-message.js
+export default Ember.Component.extend({
+  actions: {
+    sendMessage(messageType, messageText) {
+      //send message here and return a promise
+    }
+  }
+});
 ```
 
-The `send-message` component provides an input as block content to the `button-with-confirmation` component, setting
-`confirmValue`.
+We want to arrange for the action to be invoked from within `button-with-confirmation` with both arguments.
+A feature of action arguments that applies to this situation is that they _curry_.
+In general (not just in the context of Ember), this refers to the idea that a function with multiple arguments can be expressed as a sequence of functions each taking one argument and returning the next function in the sequence that takes the next argument,
+and so on.
+Evaluating the last function produced in this way is equivalent to evaluating the original function with all of its arguments.
 
-```app/templates/components/send-message.hbs
-{{#button-with-confirmation
-    text="Click to send your message."
-    onConfirm=(action "sendMessage" "info")
-    as |confirmValue|}}
-  {{input value=confirmValue}}
-{{/button-with-confirmation}}
-```
+For our purposes, currying means that when we pass the action to a component, if we provide the action helper with a subset of arguments (e.g. by providing the value of `messageType` when we pass `sendMessage` to `button-with-confirmation`), the result of the helper (in our case `onConfirm`) is a function that can be invoked within the component javascript file with just the remaining arguments,
+in our case `messageText`.
```

Comment:
From a reader's point of view, do we even care what currying is?  It's a valuable concept to know about, but I'm not sure we need to introducing that concept here (as @locks mentioned earlier)